Makefile

```
@@ -1,11 +1,12 @@
 # Ensure Make is run with bash shell as some syntax below is bash-specific
 SHELL:=/usr/bin/env bash
 
-# Define registries
-STAGING_REGISTRY ?= gcr.io/k8s-staging-cluster-api
-
 IMAGE_NAME ?= cluster-api-byoh-controller
 TAG ?= dev
+RELEASE_DIR := out
```

`out` directory mostly store the output of the go coverage tool. `_dist` will be more appropriate? As it clearly indicates the distribution of artifacts. (don't forget to add it to `.gitignore`)
```suggestion
RELEASE_DIR := _dist
```